modules/core/include/opencv2/core/base.hpp

```diff
@@ -79,18 +79,18 @@ enum Code {
     BadImageSize=              -10,  //!< image size is invalid
     BadOffset=                 -11,  //!< offset is invalid
     BadDataPtr=                -12,  //!<
-    BadStep=                   -13,  //!<
+    BadStep=                   -13,  //!< image step is wrong
     BadModelOrChSeq=           -14,  //!<
-    BadNumChannels=            -15,  //!<
+    BadNumChannels=            -15,  //!< bad number of channels
```

Comment:
Probably it is "not wrong", neither "invalid" or "bad" - current  value is usually "not supported"/"unsupported" by current implementation.